scrapers/koji.py

```diff
@@ -218,9 +218,12 @@ def update_neo4j(self, builds):
             xml_root = ET.fromstring(task_dict['request'])
             commit_hash = None
             for child in xml_root.iter('string'):
+                # Avoids incorrect creation of Commits
+                if '.tar' in child.text:
+                    commit_hash = None
+                    break
                 if child.text and child.text.startswith('git'):
                     commit_hash = child.text.rsplit('#', 1)[1]
-                    break
```

Comment:
You want to keep this break statement.

Reply:
I removed the break statement because what if our other condition to check if there's a .tar is after we extract the hash (which isn't really hash)

Reply:
Oh I see... I think there's probably a better way to do this. Let me dig into it.

Reply:
Another solution I can think of is after extracting the hash, check if it has special characters like '/' or '-'. Or if there's a specified length of the hash then we can check for that too. WDYT @mprahl ?